app/controllers/admin/subscriptions_controller.rb

```diff
@@ -28,7 +28,8 @@ def index
         end
         format.json {
           render_as_json @collection, ams_prefix: params[:ams_prefix],
-          payment_method_tags: payment_method_tags_by_id }
+                                      payment_method_tags: payment_method_tags_by_id
+        }
```

Comment:
Third time writing a comment. You change it so fast!

This is better. It would have been even better to put parenthesis around the many parameters of this method call. And multi-line blocks are often better identified with `do ... end`.

Reply:
Haha sorry! Good points, I wholeheartedly agree. I should have known better than to start down the rabbit-hole...